packages/classic_terraswap/Cargo.toml

```diff
@@ -17,16 +17,16 @@ documentation = "https://docs.terraswap.io"
 backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-cw20 = { version = "0.8.0" } 
-cosmwasm-storage = { version = "0.16.0" }
-terra-cosmwasm = { version = "2.2.0" }
-cosmwasm-std = { version = "0.16.7" }
+cw20 = { version = "1.0" } 
+cosmwasm-storage = { version = "1.1" }
+classic-bindings = { version = "0.1.1" }
+cosmwasm-std = { version = "1.1" }
```

Comment:
It seems that the migration to go to cosmwasm 1.1 is lacking. I think 1.0.0 would be fine

https://github.com/CosmWasm/cosmwasm/blob/main/MIGRATING.md

Reply:
https://github.com/terraswap/classic-terraswap/pull/14/commits/0f8a6a26c9a49e65dbde7b6a53892a053c76fbcf